core/google/cloud/streaming/transfer.py

```diff
@@ -370,18 +370,30 @@ def initialize_download(self, http_request, http):
 
         :type http: :class:`httplib2.Http` (or workalike)
         :param http: Http instance for this request.
+
+        :type range_bytes: tuple
+        :param range_bytes: Optional. Range of bytes to download.
```

Comment:
A couple things here, can you change...
```diff
+ (Optional)
- Optional
```

Also, we might want to link to the docs you referenced incase someone wants to dig into how this works behind the scenes.

```
https://cloud.google.com/storage/docs/xml-api/reference-headers#range
```